server/src/main/java/org/activityinfo/service/blob/GcsBlobFieldStorageService.java

```diff
@@ -43,23 +40,31 @@
 
     public static final int MAX_BLOB_LENGTH_IN_MEGABYTES = 100;
 
-    private final String bucketName;
+    private String bucketName;
     private AppIdentityService appIdentityService;
 
     @Inject
     public GcsBlobFieldStorageService(DeploymentConfiguration config) {
         this.bucketName = config.getBlobServiceBucketName();
-        appIdentityService = DeploymentEnvironment.isAppEngineDevelopment() ?
+
+        this.appIdentityService = DeploymentEnvironment.isAppEngineDevelopment() ?
                 new DevAppIdentityService(config) : AppIdentityServiceFactory.getAppIdentityService();
 
-        LOGGER.info("Service account: " + appIdentityService.getServiceAccountName());
+        try {
+            LOGGER.info("Service account: " + appIdentityService.getServiceAccountName());
+        } catch (ApiProxy.CallNotFoundException e) {
+            // ignore: fails in local tests, bug in LocalServiceTestHelper?
+        }
     }
 
+    @GET
+    @Path("{blobId}/blob_url")
```

Comment:
I believe all of our REST urls use camel case, while this end point uses underscores. Can we consistenly use camelCase?